torch/csrc/jit/codegen/cuda/lower_alias_memory.cpp

```diff
@@ -137,7 +137,8 @@ class AllocateReuseModifier {
 
     // Assume the first argument contains the primary variable
     // Follow path along point-wise operations
-    if (first_tv_input != nullptr) {
+    if (first_tv_input != nullptr &&
```

Comment:
nitpick: maybe update the comment as well since the logic is updated.

Reply:
This is a bug fix rather than a change to the core logic.